assets/js/components/ClusterDetails/ChecksResultOverview.jsx

```diff
@@ -0,0 +1,72 @@
+import { format } from 'date-fns';
+import {
+  EOS_CHECK_CIRCLE_OUTLINED,
+  EOS_ERROR_OUTLINED,
+  EOS_WARNING_OUTLINED,
+} from 'eos-icons-react';
+import React from 'react';
+
+const uiForResult = {
+  passing: {
+    color: 'green',
+    component: EOS_CHECK_CIRCLE_OUTLINED,
+    copy: 'Passed',
```

Comment:
In the main dashboard we use the word `Passing`
By the way, doesn't `copy` look strange? I would maybe use `text` there.

Reply:
done